apps/photos/src/pages/gallery/index.tsx

```diff
@@ -249,6 +255,9 @@ export default function Gallery() {
     const [archivedCollections, setArchivedCollections] =
         useState<Set<number>>();
 
+    const [selectedDates, setSelectedDates] = useState([]);
+    const [unselectedDates, setUnselectedDates] = useState([]);
```

Comment:
[Question]
I know the PR is still in draft but I feel we don't need both `selectedDates` and `unselectedDates` as the latter can be derived from the former. Please correct me if I am wrong.

Reply:
My apologies, I used confusing variable names. Fixed it in my latest commit. Thanks!